prod-requirements.txt

```diff
@@ -3,6 +3,6 @@ gunicorn==19.9.0
 raven==6.10.0
```

Comment:
Oh that's an old one 😄 Not sure it supports Django 4.2, but maybe it's fine... Would be nice to see if we can switch to its successor, sentry-sdk.